### PR TITLE
Proxy setHeader from rawConnection

### PR DESCRIPTION
**Pull request by gcoonrod**
This PR proxies the `setHeader` function from a http response object so that actions running on the web server can use `data.connection.setHeader` instead of `data.connection.rawConnection.res.setHeader`.

### CODE REVIEW

**Comment by evantahler** [comment]
I think your tests are good, but the method declaration belongs as part of the web server, not the generic case.  Then, you don't need the conditional. 

Perhaps what we should do is allow `server.buildConnection()` to accept `customMethods` which can be passed in.  Then, in the generic server, `api.GenericServer.prototype.buildConnection` takes all the custom methods and applies them to the connection as it is built.  This way, we can do some checking to ensure that we aren't overriding an existing method/property in a common way.

**Comment by gcoonrod** [comment]
@evantahler Ready for round two!

**Comment by evantahler** [comment]
I like it!
Now please add a Pull request to https://github.com/actionhero/www.actionherojs.com with the documentation update.

**Comment by evantahler** [comment]
Oh, and also the tests should pass :D

**Comment by gcoonrod** [comment]
Sorry this is taking so long. Hope to have time in the coming week to work on it!

File: servers/web.js [hunk 1]
```diff
@@ -342,7 +342,10 @@ const initialize = function (api, options, next) {
         id: fingerprint + '-' + uuid.v4(),
         fingerprint: fingerprint,
         remoteAddress: remoteIP,
-        remotePort: remotePort
+        remotePort: remotePort,
+        customMethods: function (connection) {
+          connection.setHeader = res.setHeader.bind(res)
```
Comment (evantahler) [inline servers/web.js]: shouldn't this be `connection.rawConnection.res.setHeader.bind(res)`

**Pull request closed without merging**